packages/amplify-codegen/src/utils/GraphQLStatementsFormatter.js

```diff
@@ -0,0 +1,72 @@
+const prettier = require('prettier');
```

Comment:
I suggest a unit test (snapshot) for this class with a predefined statements for the lint overrides/header comments under each language.